src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java

```diff
@@ -146,8 +146,8 @@ private int rule0(char s[], int len) {
 
     if (len > 6 && (endsWith(s, len, "σκαγιου"))
         || endsWith(s, len, "σκαγιων")
-        || endsWith(s, len, "ολογιου")
-        || endsWith(s, len, "ολογιων")
+        // || endsWith(s, len, "ολογιου")
+        // || endsWith(s, len, "ολογιων")
```

Comment:
Please remove the commented lines completely. We have git history to look for them if we want.